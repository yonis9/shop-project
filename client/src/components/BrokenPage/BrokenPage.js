import React from 'react';

import { 
    ErrorImageOverlay,
    ErrorImageContainer,
    ErrorImageText
} from './BrokenPage.styles';

const BrokenPage = () => (
    <ErrorImageOverlay>
        <ErrorImageContainer imageUrl='https://i.imgur.com/yW2W9SC.png'/>
        <ErrorImageText>This Page is Broken</ErrorImageText>
    </ErrorImageOverlay>
)


export default BrokenPage;