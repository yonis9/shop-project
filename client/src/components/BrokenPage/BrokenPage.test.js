import React from 'react';
import { shallow } from 'enzyme';
import BrokenPage from './BrokenPage';

it('should rendered brokenPage component', () => {
    expect(shallow(<BrokenPage />)).toMatchSnapshot();
});