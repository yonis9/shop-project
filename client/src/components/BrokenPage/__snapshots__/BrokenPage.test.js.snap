// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rendered brokenPage component 1`] = `ShallowWrapper {}`;
